### PR TITLE
Automatically adjust css for cards containing child cards

### PR DESCRIPTION
**Pull request by cyril-sf**

### CODE REVIEW

**Comment by cyril-sf** [comment]
@hjdivad Not sure how to write this test. I have the impression that it takes time for the browser to take the injected rule into account. Is there an event I could use to wait for the DOM to get updated?

File: lib/consumers/render_consumer.js [hunk 1]
```diff
@@ -3,9 +3,29 @@ Conductor.renderConsumer = function(promise, card) {
   var domInitialized = false;
 
   options.events.render = function(args) {
-    if(!domInitialized && card.initializeDOM) {
+    if(!domInitialized) {
+      if(card.renderMaximized) {
```
Comment (hjdivad) [inline lib/consumers/render_consumer.js]: Why do we have this flag?  Why not just always include the reset css?

**Comment by hjdivad** [comment]
@Cyril-sf There shouldn't be a timing issue if you create the style tag in `<HEAD>`.  You should be able to just assert the value of `DomUtils.getComptuedStyleProperty(document.body, 'padding')`.

File: lib/consumers/render_consumer.js (hunk 1 above)
Comment (cyril-sf) [inline lib/consumers/render_consumer.js]: My understanding was that the issue exists mostly for cards with child cards.

Comment (hjdivad) [inline lib/consumers/render_consumer.js]: The padding issue exists for everyone.  The `iframe` issue affects child cards.  It seems simpler to just always include it.

**Comment by cyril-sf** [comment]
@hjdivad I've updated the code

File: lib/consumers/render_consumer.js [hunk 2]
```diff
@@ -2,10 +2,33 @@ Conductor.renderConsumer = function(promise, card) {
   var options = Object.create(card.options);
   var domInitialized = false;
 
+  function resetCSS() {
+    var newScript = document.createElement('style');
```
Comment (hjdivad) [inline lib/consumers/render_consumer.js]: `newScript` -> `styleTag`?

**Comment by hjdivad** [comment]
@Cyril-sf This looks good to merge except the var in `resetCSS` is a style tag not a script, no?

**Pull request merged**